frontend/app/angular4-modules.ts

```diff
@@ -542,16 +544,23 @@ import {WorkPackageRelationsCreateComponent} from "core-components/wp-relations/
 
     // External query configuration
     ExternalQueryConfigurationComponent,
+
+    // Main menu
+    MainMenuResizerDirective
   ]
 })
 export class OpenProjectModule {
   constructor(private upgrade:UpgradeModule) {
   }
 
   // noinspection JSUnusedGlobalSymbols
-  ngDoBootstrap() {
+  ngDoBootstrap(appRef:ApplicationRef) {
     // Already done in openproject-app.ts
     // this.upgrade.bootstrap(document.body, ['openproject'], {strictDi: false});
+
+    if (document.getElementsByTagName('main-menu-resizer').length > 0) {
+      appRef.bootstrap(MainMenuResizerDirective);
```

Comment:
Why didn't you use `downgradeComponent` in this case?

Reply:
I understand that too little. Why should we downgrade to AngularJS the main-menu-resizer is an Angular entry component and does not run within AngularJS? Probably I don't understand it enough.